src/tasks/cleanup.ts

```diff
@@ -42,7 +43,8 @@ const HEADER = '\u001B[39m\u001B[94m[CACHE CLEANUP]\u001B[39m\u001B[90m';
 export default class extends Task {
 
 	public run() {
-		const OLD_SNOWFLAKE = Util.binaryToID(((Date.now() - THRESHOLD) - EPOCH).toString(2).padStart(42, '0') + EMPTY);
+		const binary = ((Date.now() - THRESHOLD) - EPOCH).toString(2).padStart(42, '0') + EMPTY;
+		const OLD_SNOWFLAKE = new Snowflake(binary).toString();
```

Comment:
We have two options:

## 1. Drop the dependency and do the raw maths ourselves

Actually, we don't need this anymore, binaryToID was used for Node.js v10, but we're using Node.js v14 in Skyra (and we will probably bump it soon).

```typescript
		const millisecondsSinceEpoch = BigInt(Date.now()) - EPOCH - THRESHOLD;
		const OLD_SNOWFLAKE = ((millisecondsSinceEpoch << 22n) + 0b0000100000000000000000n).toString();
```

This new code generates (from message ID `724723793200939168`), this one: `724716442389446656`, which is roughly 30 minutes earlier.

This change will need to change lines 33 and 34, the former to do `BigInt(Time.Minute * 30)` and the latter to do `1420070400000n`.

Once this change is done, you can safely drop `@klasa/snowflake` from dependencies.

## 2. We use Snowflake.generate

```typescript
		const OLD_SNOWFLAKE = Snowflake.generate(Date.now() - THRESHOLD);
```

Doesn't need any further modification except dropping the `EPOCH` constant.